src/Compute/docs/hybrid-2019-03-01/Az.Compute.md

```diff
@@ -109,14 +109,14 @@ Create or update an image.
 ### [New-AzSnapshot](New-AzSnapshot.md)
 Creates or updates a snapshot.
 
-### [New-AzVM](New-AzVM.md)
-The operation to create or update a virtual machine.
-
 ### [New-AzVMExtension](New-AzVMExtension.md)
 The operation to create or update the extension.
 
 ### [New-AzVmss](New-AzVmss.md)
-Create or update a VM scale set.
+
+
+### [New-AzVmss](New-AzVmss.md)
```

Comment:
@markcowl something is wrong with the changes in this file -- `New-AzVM` isn't included and there's two entries for `New-AzVmss`